app/assets/stylesheets/modules/homepage.scss

```diff
@@ -1,16 +1,3 @@
-.bento-panels {
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: space-between;
-
-  .card {
-    margin-left: auto;
-    margin-right: auto;
-    max-width: 24%;
-    min-width: 24%;
-  }
-}
-
 .homepage-panels {
```

Comment:
I believe this should be removed too.

Reply:
The more search tools partial is used in the index page, and these styles are still used there.  I didn't touch the index page so didn't include these for removal.  I can remove them if you would like (if they are already being removed elsewhere for search)